src/utils/CaptainConstants.ts

```diff
@@ -54,6 +54,21 @@ const configs = {
     proApiDomains: ['https://pro.caprover.com'],
 
     analyticsDomain: 'https://analytics-v1.caprover.com',
+
+    certbotImageName: 'caprover/certbot-sleeping:v1.6.0',
+
+    certbotCertCommand: undefined as CertbotCertCommandRule[] | undefined,
+}
+
+export interface CertbotCertCommandRule {
+    /**
+     * Matches both *.<domain> and <domain>, use '*' to match all domains
+     */
+    domain: string;
+    /**
+     * The Certbot command to execute, in Docker exec form, uses `${domain}` as the placeholder for the actual domain name
+     */
+    command?: string[];
```

Comment:
Let's use `command?: string`, when passing to exec, we can do `command.trim().split(' ')`

This will make it much easier to override and read.

Reply:
Sounds good! We can use [shell-quote](https://www.npmjs.com/package/shell-quote#parse-with-an-environment-variable) to parse the command string before passing it to exec. It's more robust than using `String.split`.